### ISSUE TITLE
Feature request and few issues.

### ISSUE THREAD

**Issue by Garfius** [dhansel/VersaTerm#11]
Hello, this project is awesome, i have just build one of yours, and works pretty nice, but soldering the HDMI was almost impossible.

Just wondering/asking wether this project can run on a '[rp2040-piZero](https://www.waveshare.com/rp2040-pizero.htm)', which already has the HDMI, so no soldering will be needed.

I just bought one rp2040-piZero, so i am ready for testing ;)

By the way... The issues are:

1. I use PS2 keyboard, home and end keys does not send proper escape codes.
2. HDMI output does not detect a [1024*600 display which uses Realtek rtd2660](https://es.aliexpress.com/item/1005002863549499.html).

![image](https://github.com/dhansel/VersaTerm/assets/12560815/74371f0d-33bc-4b71-bb37-b5ba2db126f9)

**Comment by Garfius** [comment]
Ok, here trying to make it work on a RP2040-PiZero.

Changed speed at 9600 and all the ping, USB keyboard, uart0 removed VGA option.

I included PIN_USBMODE at pins.h because this board does not have this feature, also pin 24 is used for 'HDMI D1+', now works using duPont cable, might change it later.

I've got a red lines problem, compiled using Debug i can only see few maybe 2 lines, on Release i see what following photo shows:

https://github.com/Garfius/VersaTerm-PiZero/blob/main/software/screenshots/rp2040-piZero-alpha.jpg

There is also the binary available: 
https://github.com/Garfius/VersaTerm-PiZero/blob/main/rp2040-piZero-alpha.uf2

Any idea?

HELP
